README.md

```diff
@@ -149,7 +149,7 @@ the Mbed TLS source directory, use:
 If you want to change `CC` or `CFLAGS` afterwards, you will need to remove the
 CMake cache. This can be done with the following command using GNU find:
 
-    find . -iname '*cmake*' -not -name CMakeLists.txt -exec rm -rf {} +
+    find . -name .git -prune -o -iname '*cmake*' -not -name CMakeLists.txt -exec rm -rf {} +
```

Comment:
This is documentation, not a script. What if the user isn't using git, but is using some other hidden directory that has similar problems? Wouldn't it be better to leave the original command and add a comment to say 'by the way this is destructive, and be sure you're not deleting hidden files that you need, such as in `.git` directories?'

Reply:
Good point. I'll leave the command as is (optimising for the most common case) but add a comment that it should be adapted in other cases.